docs/usage-guide/validators.md

```diff
@@ -45,9 +45,9 @@ enter an email address again (hopefully a valid one this time).
 ```
 
 [^1]:
-    Technically, the type alias for a `Validator` is `Callable[[str, Answers], Union[ValidationResponse, Optional[str]]]` - the difference
+    Formerly, the type alias for a `Validator` was `Callable[[str, Answers], Union[ValidationResponse, Optional[str]]]` - the difference
     from the documentation above is that a `Validator` can return either a `ValidationResponse` *or* `Optional[str]`. Returning `Optional[str]`
```

Comment:
Suggested improvement:

```suggestion
    Formerly, the type alias for a `Validator` was `Callable[[str, Answers], Union[ValidationResponse, Optional[str]]]` -
    in which a `Validator` can return either a `ValidationResponse` *or* `Optional[str]`. Returning `Optional[str]`
```

Reply:
I ended up just removing this. Thanks for the suggestion!